package.json

```diff
@@ -23,11 +25,16 @@
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.27.5",
     "hint": "^7.1.3",
+    "html-webpack-plugin": "^5.5.0",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
     "stylelint-csstree-validator": "^1.9.0",
     "stylelint-scss": "^3.21.0",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.1"
+  },
+  "dependencies": {
+    "css-loader": "^6.7.3",
+    "style-loader": "^3.3.1"
```

Comment:
- [ ] Kindly install **`webpack-dev-server`** as part of your dependencies in order for the app to start properly without asking user to install it.` webpack-dev-server` is a CLI-based tool for starting a static server for your assets. ℹ️ 

> I'm sure **npm i webpack-dev-server** will do the trick. 😄